virttest/utils_disk.py

```diff
@@ -309,7 +309,7 @@ def update_windows_disk_attributes(session, dids, timeout=120):
             if status != 0:
                 logging.error("Can not clear readonly bit: %s" % output)
                 return False
-        pattern = "DISK %s.*Offline" % did
+        pattern = "Status.*Offline"
         if re.search(pattern, details, re.I | re.M):
```

Comment:
I think we can discard 'pattern'.
```python
    if re.search("Status.*Offline", details, re.I | re.M):
```

Reply:
Agree with you, I will update. Thank you.